specification/eventgrid/data-plane/Microsoft.KeyVault/stable/2018-01-01/KeyVault.json

```diff
@@ -312,6 +312,40 @@
           "type": "number"
         }
       }
+    },
+     "KeyVaultVaultAccessPolicyChangedEventData": {
```

Comment:
Formatting is off, please run "prettier" to fix (see failed CI job for more info).

Reply:
Updated